packages/expo-permissions/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### 🎉 New features
 
+- If permission is not recognized, show the correct expo package to link. ([#8546])(https://github.com/expo/expo/pull/8046) by [@jarvisluong](https://github.com/jarvisluong)
```

Comment:
Looks like you have outdated changelog on your branch 🤔 Those bug fixes below have already been published in 8.2.0

Reply:
That's right, my master is 35 commits behind the current master, should I rebase now?

Reply:
Yeah I think so, sorry for that 🙂 Unfortunately you may get some issues with rebasing because one of the commit you didn't have is https://github.com/expo/expo/commit/e166a8d8b72263a5ab75f04871e7a5d2da9729e9 which replaced `react-native-unimodules` as a submodule — we've moved it into the monorepo. Git sometimes has some troubles to resolve such changes right 😞

Reply:
this should be fine now 😅